crates/bevy_ecs/macros/src/lib.rs

```diff
@@ -180,6 +180,93 @@ pub fn derive_bundle(input: TokenStream) -> TokenStream {
     })
 }
 
+#[proc_macro_derive(IterEntities, attributes(iter_entities))]
```

Comment:
I think it worth documenting this derive and add a doc example. Maybe add it to the trait?

Reply:
:+1: I'll add that to my to-do list